src/ring_theory/unique_factorization_domain.lean

```diff
@@ -1127,3 +1128,12 @@ noncomputable def unique_factorization_monoid.to_gcd_monoid
   .. ‹normalization_monoid α› }
 
 end
+
+instance polynomial.unique_factorization_monoid
```

Comment:
I'm confused... In ordinary maths, you don't need the normalization assumption, right? So why do we need it in Lean?

Reply:
The reason is that I'm using the `gcd_monoid` structure to prove the equivalence between irreducible and prime, and `gcd_monoid` still needs normalization.

Every `unique_factorization_monoid` can be made into a `normalization_monoid`, but the decision is highly arbitrary and I haven't added it to mathlib yet.
Do you think I should include that in this PR?

Reply:
Aha, I see. I would very much like to see that in mathlib. But this PR is big enough already.
How hard do you think it is to add that fact?

Reply:
I don't think it'd be toooo hard, but the mathlib implementation will probably be at least a bit fiddly. I think I'll separate all the UFD facts into a second PR, which hopefully won't be too long.